pkg/apis/core/validation/controllerregistration.go

```diff
@@ -41,6 +41,12 @@ var availableExtensionStrategies = sets.New(
 	string(core.AfterKubeAPIServer),
 )
 
+var availableExtensionStrategiesForReconcile = sets.New(
+	string(core.BeforeKubeAPIServer),
+	string(core.AfterKubeAPIServer),
+	string(core.AfterWorker),
+)
```

Comment:
What about the following? This would ensure we don't miss adding a new strategy to this new set as well.
```go
var availableExtensionStrategiesForReconcile = availableExtensionStrategies.Clone().Insert(
	string(core.AfterWorker),
)
```